src/components/pages/rules/flyouts/ruleStatusFlyout.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+import { RuleStatusContainer } from './ruleStatus';
+import Flyout from 'components/shared/flyout';
+
+export const RuleStatusFlyout = ({ t, onClose, rules }) => (
+  <Flyout.Container>
+    <Flyout.Header>
+      <Flyout.Title>{t('rules.flyouts.statusTitle')}</Flyout.Title>
+      <Flyout.CloseBtn onClick={onClose} />
+    </Flyout.Header>
+    <Flyout.Content>
+      <RuleStatusContainer onClose={onClose} rules={rules} />
```

Comment:
The flyout and the rule status form should be the same component. The flyout is not useful factored out and the rule status is useful factored out (no one else is using it). They should be merged together. We can make it a todo to not block this PR though.